library/core/src/hash/mod.rs

```diff
@@ -503,10 +491,7 @@ pub trait BuildHasher {
 /// let hash_map = HashMap::<u32, u32, MyBuildHasher>::default();
 /// ```
 ///
-/// [`BuildHasher`]: trait.BuildHasher.html
-/// [`Default`]: ../default/trait.Default.html
-/// [method.default]: #method.default
-/// [`Hasher`]: trait.Hasher.html
+/// [method.default]: BuildHasherDefault::default
```

Comment:
This is failing. Should I try to resolve it as `Default::default` instead or just leave the original link?

Reply:
If you wait a week until the beta branch, it will be fixed by #74489.

Reply:
Hmm, doesn't the beta branch happen today? https://rust-lang.zulipchat.com/#narrow/stream/122651-general/topic/When.20is.20the.20cutoff.20for.20the.20beta.20release.3F

Reply:
Not until tomorrow: https://forge.rust-lang.org/release/process.html#master-bootstrap-update-t-1-day-wednesday

Reply:
Thanks! I love this line:

> Bask in your success.